### PR TITLE
Updates to 1.14.2

### PR DESCRIPTION
**Pull request by elena-kolevska**

### CODE REVIEW

**Review by mikeee — APPROVED** [review]

File: release_deploy_timeline.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # deploy-longhaul-release timeline
 
+- 2024-08-08 - Updated dapr version to 1.14.0-rc.8
```
Comment (JoshVanL) [inline release_deploy_timeline.md:3]: Should this not be `1.14.0-rc.11`?

**Review by JoshVanL — CHANGES_REQUESTED** [review]

File: release_deploy_timeline.md (hunk 1 above)
Comment (elena-kolevska) [reply release_deploy_timeline.md:3]: No, this is for the 8th of August, when we moved to rc8. Once we move to rc11 I'll update the timeline file with the correct date.

Comment (mikeee) [reply release_deploy_timeline.md:3]: > No, this is for the 8th of August, when we moved to rc8. Once we move to rc11 I'll update the timeline file with the correct date.

Just a thought, is a downside of updating notes post-changes a cause for potential redeployment of resources?

Comment (elena-kolevska) [reply release_deploy_timeline.md:3]: Ah no, this note is just for our own convenience, so we have a single place to look at when we're looking at the graphs on a timeline, and we don't have to go and look which PRs were merged when.

**Comment by elena-kolevska** [comment]
Updated to 1.14

**Comment by elena-kolevska** [comment]
We can close this PR since https://github.com/dapr/test-infra/pull/235 was merged

**Pull request closed without merging**